src/index.js

```diff
@@ -1,42 +1 @@
-import defaultLog from "./defaultLog"
-
-export default function(options) {
-  options = options || {}
-  options.log = typeof options.log === "function" ? options.log : defaultLog
-
-  return function(app) {
-    return function(initialState, actionsTemplate, view, container) {
-      function enhanceActions(actions, prefix) {
-        var namespace = prefix ? prefix + "." : ""
-        return Object.keys(actions || {}).reduce(function(otherActions, name) {
-          var namedspacedName = namespace + name
-          var action = actions[name]
-          otherActions[name] =
-            typeof action === "function"
-              ? function(data) {
-                  return function(state, actions) {
-                    var result = action(data)
-                    result =
-                      typeof result === "function"
-                        ? result(state, actions)
-                        : result
-                    options.log(
-                      state,
-                      { name: namedspacedName, data: data },
-                      result
-                    )
-                    return result
-                  }
-                }
-              : enhanceActions(action, namedspacedName)
-          return otherActions
-        }, {})
-      }
-
-      var enhancedActions = enhanceActions(actionsTemplate)
-
-      var appActions = app(initialState, enhancedActions, view, container)
-      return appActions
-    }
-  }
-}
+export { withLogger } from "./withLogger"
```

Comment:
Why do you need this re-export file, why not just include logger implementation here or remove this file at all?
It is so annoying when you click on import in IDE and you see such file when you expect to see implementation.

Reply:
Agreed. 💯

Reply:
Guess I've got a bad habit of always using `index.js` for my exports to the outside world and naming other files by what they export 😬 

Changed.

Reply:
@okwolf What if you rename it to `index.js`, but export `withLogger` from it? Like how we do in hyperapp itself. Would that be better? I think so.

Reply:
@jorgebucaran take a look at my newest commit, I think you'll approve.